models/migration.go

```diff
@@ -46,11 +48,22 @@ func migration() {
 	// 创建初始存储策略
 	addDefaultPolicy()
 
+	// 获取第一个存储策略
+	policy, err := GetFirstPolicy()
+	if err != nil {
+		util.Log().Panic(fmt.Sprintf("Failed to get frist policy: %v", err))
+	}
+
 	// 创建初始用户组
-	addDefaultGroups()
+	addDefaultGroups(policy.ID)
+
+	group, err := GetFristGroup()
```

Comment:
How can you make sure the first group is the admin group?

Reply:
I see. But I think the follow code is  a better way, instead of the undefined primary key.
`func addDefaultGroups() {
      	_, err = GetGroupByID(2)
	if gorm.IsRecordNotFoundError(err) {
                   group := Group{
                           Model: gorm.Model{ID:2}
                           ....
                   }
                   DB.Create(group)
        }
}`